rfc6962-bis.xml

```diff
@@ -1130,6 +1130,86 @@ entry specified by <spanx style="verb">start</spanx>.
         <t>See <xref target="verify_hash" /> for an outline of how to use a complete list of <spanx style="verb">leaf_input</spanx> entries to verify the <spanx style="verb">root_hash</spanx>.</t>
       </section>
 
+      <section title="Get Entry Number for SCT">
+	<t>
+	  Logs MAY implement this message. It is not required for correct operation of logs or their clients, but may be convenient in some circumstances, since an SCT cannot be validated with also knowing which chain it applies to. Note that mirrors can implement this even if the log they are mirroring does not.
+	</t>
+	<t>
+	  GET https://&lt;log server&gt;/ct/v2/get-entry-for-sct
+	</t>
+	<t>
+          <list style="hanging">
+            <t hangText="Inputs:">
+              <list style="hanging">
+                <t hangText="sct:">
+                  A base64 encoded <spanx style="verb">TransItem</spanx> of type <spanx style="verb">x509_sct_v2</spanx> or <spanx style="verb">precert_sct_v2</spanx> allegedly from this log.
+                </t>
+              </list>
+            </t>
+            <t hangText="Outputs:">
+              <list style="hanging">
+                <t hangText="entry:">
+		  0-based index of the log entry corresponding to the supplied SCT.
+                </t>
+              </list>
+            </t>
+            <t hangText="Error codes:">
+              <list style="hanging">
+                <t hangText="bad signature">
+                  <spanx style="verb">sct</spanx> is not signed by this log.
+		</t>
+                <t hangText="not found">
+                  <spanx style="verb">sct</spanx> does not correspond to an entry that is currently available.
+		</t>
+	      </list>
+	    </t>
+          </list>
+        </t>
+	<t>
+	  Note that any SCT with a valid signature MUST have a corresponding entry in the log, but it may not be retrievable until the MMD has passed since the SCT was issued.
+	</t>
+      </section>
+
+      <section title="Get Entry Numbers for Certificate">
+	<t>
+	  Logs MAY implement this message. It is not required for correct operation of logs or their clients, but may be convenient in some circumstances, for example if a certificate is not accompanies by an SCT yet a client wants to verify whether it has, in fact, been logged. Note that mirrors can implement this even if the log they are mirroring does not.
+	</t>
+	<t>
+	  GET https://&lt;log server&gt;/ct/v2/get-entry-for-certificate
+	</t>
+	<t>
+          <list style="hanging">
+            <t hangText="Inputs:">
+              <list style="hanging">
+                <t hangText="hash:">
+                  A base64 encoded HASH of a <spanx style="verb">TBSCertificate</spanx>.
```

Comment:
Since the log entry both for X.509 certs and precerts now has the issuer_key_hash in addition to the TBSCertificate, does it make sense to require it here as well?
May not be, just bringing it up for discussion.